src/libostree/ostree-repo-pull.c

```diff
@@ -2995,6 +3000,12 @@ _ostree_repo_remote_new_fetcher (OstreeRepo *self, const char *remote_name, gboo
       }
   }
 
+  _ostree_fetcher_set_low_speed_limit (fetcher, (low_speed_limit) ? low_speed_limit
+                                                                  : OPT_LOWSPEEDLIMIT_DEFAULT);
```

Comment:
I think we don't need the `?` here because we're already effectively setting the default below, so this can just be:

```
  _ostree_fetcher_set_low_speed_limit (fetcher, low_speed_limit);
```
right?

And same for the other two.